LeaderboardBackend/Controllers/RunsController.cs

```diff
@@ -102,6 +103,37 @@ public async Task<ActionResult<RunViewModel>> CreateRun(
         );
     }
 
+    [AllowAnonymous]
+    [HttpGet("/api/category/{id:long}/runs")]
+    [Paginated]
+    [SwaggerOperation("Gets the Runs for a Category.", OperationId = "getRunsForCategory")]
+    [SwaggerResponse(200)]
+    [SwaggerResponse(404, "The Category with ID `id` could not be found, or has been deleted. Read `title` for more information.")]
+    [SwaggerResponse(422, Type = typeof(ValidationProblemDetails))]
+    public async Task<ActionResult<ListView<RunViewModel>>> GetRunsForCategory(
+        [FromRoute] long id,
+        [FromQuery] Page page,
+        [FromQuery, SwaggerParameter(Required = false, Description = "Whether to include deleted runs. Defaults false.")] bool includeDeleted = false
+    )
+    {
+        GetRunsForCategoryResult result = await runService.GetRunsForCategory(id, page, includeDeleted);
+
+        return result.Match<ActionResult>(
+            runs => Ok(new ListView<RunViewModel>()
+            {
+                Data = runs.Items.Select(RunViewModel.MapFrom).ToList(),
+                Total = runs.ItemsTotal
+            }),
+            notFound => NotFound(
+                ProblemDetailsFactory.CreateProblemDetails(
+                    HttpContext,
+                    404,
+                    "Category Not Found."
```

Comment:
Remove the period here.

Reply:
But all other error messages of the controller end with a period.

Reply:
Mimic whatever the default behavior is.

Reply:
Title case and no period, turns out. I'll remove the periods